src/main/java/com/innobridge/ethpay/configuration/ExternalClientConfiguration.java

```diff
@@ -23,7 +23,7 @@
 public class ExternalClientConfiguration {
 
     @Bean
-    public MongoClient mongoClient(@Value("${MONGO_DATABASE_URI:}") String mongoDatabaseUri) {
+    public MongoClient mongoClient(@Value("${spring.data.mongodb.uri}") String mongoDatabaseUri) {
```

Comment:
For some reason, have to use the value from the "application.properties" file, not sure why am not able to access the ".env" file here directly, any ideas @yilengyao ?

Reply:
The .env file is for docker-compose
you have to run
`source MONGO_DATABASE_URI=...`
if you run it without docker.